next-docs/pages/guides/extending-world.mdx

```diff
@@ -1 +1,484 @@
 # Extending World
+
+On this page you learn how to modify a `World` that is already deployed to the blockchain.
+If you want to learn how to modify a `World` before it is deployed, [see the hello world page](./hello-world).
```

Comment:
Should we add one sentence about how everything below assumes there is a default template world deployed already and that we know its world address? (unless i missed that sentence?) For the "how" we can just link to the getting started guide.

Reply:
It's at line 148, but I also added some language here.